cloudferry_devlab/cloudferry_devlab/tests/utils.py

```diff
@@ -156,9 +160,13 @@ def execute_command_on_vm(self, ip_addr, cmd, username=None,
                                  warn_only=warn_only):
             try:
                 if use_sudo:
-                    return fabric_api.sudo(cmd, shell=False)
+                    output = fabric_api.sudo(cmd, shell=False)
+                    self.log.debug(output)
+                    return output
                 else:
-                    return fabric_api.run(cmd, shell=False)
+                    output = fabric_api.sudo(cmd, shell=False)
```

Comment:
Should be `fabric_api.run` here. Also make sure the logs clearly show whether the call was done with `sudo` or with `run`